src/Menu/Menu.js

```diff
@@ -125,10 +125,6 @@ class Menu extends Component<DefaultProps, AllProps, void> {
       // $FlowFixMe
       menuList.style.width = `calc(100% + ${size})`;
     }
-
-    if (this.props.onEnter) {
-      this.props.onEnter(element);
```

Comment:
Any good reasoning for a "No" without any explanation? I would like to better understand "why" you don't think we should change this behavior, and why you don't feel a Menu is a good solution (and what else you would recommend instead).